source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -288,14 +298,146 @@ operations to execute in any order:
+see the following API documentation:
 
 - `BulkWriteOptions <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/BulkWriteOptions.html>`__
 - `ordered() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/BulkWriteOptions.html#ordered(boolean)>`__
 
+.. _java-sync-client-bulk-write:
+
+Client Bulk Write Method
+------------------------
+
+When connecting to a deployment running {+mdb-server+} 8.0 or later,
+you can use the ``MongoClient.bulkWrite()`` method to write
+to multiple databases and collections in the same cluster.
+
+In the examples in preceding sections of this page, the ``MongoCollection.bulkWrite()`` method 
+takes a list of ``WriteModel`` instances in which the specified subclass of 
+``WriteModel`` represents the corresponding write operation. For example, an
+instance of ``InsertOneModel`` represents an operation to insert one document. 
+
+Similarly, the ``MongoClient.bulkWrite()`` method takes a 
+list of ``ClientNamespacedWriteModel`` instances to represent different write operations.
+However, you do not need to specify the subclass that represents the corresponding
+write operation. Instead, the ``ClientNamespacedWriteModel`` object contains different 
+methods to represent different write operations, such as ``ClientNamespacedWriteModel.insertOne()`` or 
+``ClientNamespacedWriteModel.updateOne()``.
+
+These methods take a ``MongoNamespace`` object that defines which
+database and collection to write to, and a ``Document`` object that defines
+information about the write operation. Some methods, such as ``updateOne()`` and 
+``replaceOne()``, also take a ``Filters`` object that defines the filter for the operation.
+
+The following sections describe how to use the client ``bulkWrite()`` method.
+
+Insert Example
+~~~~~~~~~~~~~~
+
+This following example shows how to use the ``bulkWrite()`` method to insert 
+two documents. One document is inserted into the ``people`` collection in a
+database named ``db``. The other document is inserted into a collection called ``things``
+in the ``db`` database.
+We use the ``MongoNamespace`` object to define the database and collection we
+want each write operation to be applied to.
+
+.. code-block:: java
+
+   MongoNamespace peopleNamespace = new MongoNamespace("db", "people");
+   MongoNamespace thingsNamespace = new MongoNamespace("db", "things");
+
+   List<ClientNamespacedWriteModel> bulkOperations = new ArrayList<>();
+
+   bulkOperations.add(ClientNamespacedWriteModel.insertOne(peopleNamespace, new Document("name", "Julia Smith")));
+   bulkOperations.add(ClientNamespacedWriteModel.insertOne(thingsNamespace, new Document("object", "washing machine")));
+
+   ClientBulkWriteResult result = client.bulkWrite(bulkOperations);
+
+.. TODO: link documentation
+
+Replace Example
+~~~~~~~~~~~~~~~
+
+The following example shows how to use the ``bulkWrite()`` method to replace
+existing documents in the ``people`` and ``things`` collections.
```

Comment:
```suggestion
existing documents in the ``people`` and ``things`` collections of the ``db`` database.
```

Reply:
changed to "db.people" and "db.things" for consistency with insert section